lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -771,5 +816,42 @@ export abstract class ClientApplication {
         return validLogoutRequest;
     }
 
+    /**
+     * Emits events by calling callback with event message
+     * @param eventType 
+     * @param interactionType 
+     * @param payload 
+     * @param error 
+     */
+    emitEvent(eventType: EventType, interactionType?: InteractionType, payload?: EventPayload, error?: EventError) {
```

Comment:
Open to discussion: wanted to have this be protected, but found we could not test for it.

Reply:
I'll find a workaround next week.

Reply:
Marking as protected doesn't end up breaking tests.